test/loader/loader.test.js

```diff
@@ -29,10 +29,10 @@ describe('loader command', () => {
         expect(existsSync(join(__dirname, loaderName))).toBeTruthy();
 
         // Test regressively files are scaffolded
-        const files = ['package.json', 'yarn.lock', 'examples', 'src', 'test', 'src/index.js', 'examples/simple/webpack.config.js'];
```

Comment:
Same reason, checking for lock file is failing at random instances, I think before the file is actually created we're running a check for it.